vertical-pod-autoscaler/hack/run-integration-locally.sh

```
@@ -133,9 +133,7 @@ fi
 export REGISTRY=${REGISTRY:-localhost:5001}
 export TAG=${TAG:-latest}
 
-rm -f ${SCRIPT_ROOT}/hack/e2e/vpa-rbac.yaml
-patch -c ${SCRIPT_ROOT}/deploy/vpa-rbac.yaml -i ${SCRIPT_ROOT}/hack/e2e/vpa-rbac.diff -o ${SCRIPT_ROOT}/hack/e2e/vpa-rbac.yaml
-kubectl apply -f ${SCRIPT_ROOT}/hack/e2e/vpa-rbac.yaml
+kubectl apply -f ${SCRIPT_ROOT}/deploy/vpa-rbac.yaml
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)